packages/matchbox/src/components/Banner/Banner.js

```diff
@@ -84,7 +89,14 @@ const Banner = React.forwardRef(function Banner(props, ref) {
 
   const dismissMarkup = onDismiss ? (
     <Box flex={['1', null, '0']} textAlign="right">
-      <StyledDismiss as="button" onClick={onDismiss} status={status} color="gray.800" type="button">
+      <StyledDismiss
+        as="button"
+        onClick={onDismiss}
+        status={status}
+        color="gray.800"
+        type="button"
+        p={size === 'small' ? 0 : '100'}
+      >
         <ScreenReaderOnly>Dismiss</ScreenReaderOnly>
         <Close size={24} />
```

Comment:
Should we also decrease this close button size a bit? might need to adjust padding too if you decrease this